radicle-httpd/src/api/v1/projects.rs

```diff
@@ -1310,4 +1310,34 @@ mod routes {
             )
         );
     }
+
+    #[tokio::test]
+    async fn test_blob_stream() {
+        use crate::api::json::blob_stream;
+        use futures_util::StreamExt;
+
+        let tmp_dir = tempfile::tempdir().unwrap();
+        std::fs::create_dir_all(tmp_dir.path()).unwrap();
```

Comment:
is this needed? doesn't `tempfile::tempdir()` already create the dir for you?

Reply:
You need to call `path` to get the `&Path` out. `tempdir` gives you back a type `TmpDir`.

Reply:
I will double check.

Reply:
You are correct. No need to create the dir (again). I added it as for some reason the dir was not created. Maybe there were some accidental mistakes I didn't notice. Now it's good 👍 . Thanks.